hack/update-codegen.sh

```diff
@@ -86,6 +86,7 @@ grafana:codegen:lsdirs() {
 grafana::codegen:run pkg
 grafana::codegen:run pkg/apimachinery
 grafana::codegen:run pkg/aggregator
+grafana::codegen:run apps/dashboard/pkg
```

Comment:
I've had to add this extra dir to make sure we codegen stuff like openapi and conversions.

I've also noticed that we don't do this for playlists / investigations / other apps – should we?

CC @toddtreece

Reply:
> this seems fine to me for now, but i thought the app SDK codegen handled openapi

Unfortunately not at the moment and it's a bit of a problem when you _don't_ use CRDs, e.g. current OpenAPI code produced for dashboards v2 is lossy (because it goes CUE -> Go -> OpenAPI but e.g. Go cannot represent enums or disjunctions properly), CC @IfSentient @K-Phoen 

> would it make sense to add an option for conversion gen as well

You mean to app SDK?

Reply:
> You mean to app SDK?

yep, but i assumed the SDK was still handling openapi codegen